v2/pkg/operator/local_stored_collector.go

```diff
@@ -237,15 +230,21 @@ func (o *LocalStorageCollector) prepareM2DCopyBatch(log clog.PluggableLoggerInte
 	for _, relatedImgs := range images {
 		for _, img := range relatedImgs {
 			imgName := img.Image
-			src := ""
+			var src string
+			var dest string
 			if !strings.Contains(src, "://") { // no transport was provided, assume docker://
```

Comment:
nit: could you put the comment above - its just a matter of style

Reply:
Done